exampleCourse/questions/demo/multiAnswerMC/question.html

```diff
@@ -0,0 +1,31 @@
+<pl-question-panel>
+    <p>
+        This example question shows how we can use the Coverage (COV) grading scheme in a
+        multiple-choice question where only one option is correct, but multiple options can be
+        selected. In this question, COV grading gives a nonzero amount of credit to any valid
+        submission that selects the correct option. Notice also that at most 3 options can be
+        selected; we accomplish this by setting <code>max-select="3"</code> in the <code>pl-checkbox</code>
+        attributes.
+    </p>
+</pl-question-panel>
+
+<div class="card my-2">
```

Comment:
We are trying to avoid bootstrap markup in example questions, because instructors tend to copy it unnecessarily. In this case I think it would be fine to just delete all the `card` markup.